Legacy/PlayFab/Multiplayer.api.json

```diff
@@ -3286,16 +3346,36 @@
         {
           "name": "ExcludeFacebookFriends",
           "description": "Controls whether this query should link to friends made on the Facebook network. Defaults to false",
+          "deprecation": {
+            "ProposedAfter": "10/14/2022",
+            "DeprecatedAfter": "12/14/2022",
+            "ObsoleteAfter": "04/24/2023",
```

Comment:
what do "DeprecatedAfter" and "ObsoleteAfter" mean for a client caller

do either imply that at some point someone calling FindFriendLobbies with one of these fields will get a failure response? that is pretty concerning to me since titles have already shipped with access to these APIs so randomly breaking them feels like a potential for a really bad customer experience. In the past we've basically operated under the principle that once a client ships we cannot randomly break them from the service.

apologies for lack of context here, from the client team I haven't had to interact with these specs before

Reply:
Those values control how the API is presented in the SDK. In our SDK generator we have logic that will potentially add attributes (depending on language) and eventually just omit the API from the SDK once specific dates have passed. Do be aware, though, that just because the obsolete date has passed, doesn't mean the API stops working, instead, it just stops showing up in new versions of SDKs that use the PlayFab SDK generator.

Reply:
ah so it's an (eventual) compile time break in the SDK and not a runtime break in the service.

I guess that's ok since it's expected that all users of the various SDKs are building from source right? there's no opportunity for an ABI break like that to not be caught if someone (for example) transparently update their SDK binary but not whatever assets are used to compile against (thinking in C++ terms I'm thinking about header files and DLLs)

thanks for clarifying :)